src/Microsoft.VisualStudio.Composition/Microsoft.VisualStudio.Composition.csproj

```diff
@@ -12,26 +12,18 @@
       <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
     </ProjectReference>
   </ItemGroup>
-  <ItemGroup Condition=" '$(TargetsDesktop)' == 'true' ">
-    <Reference Include="System.ComponentModel.Composition" />
-    <PackageReference Include="System.Threading.Tasks.Dataflow" Version="4.5.24" />
-  </ItemGroup>
-  <ItemGroup Condition=" '$(TargetsDesktop)' != 'true' ">
+  <ItemGroup>
+    <PackageReference Include="System.ComponentModel.Composition" Version="4.5.0" />
     <PackageReference Include="System.Threading.Tasks.Dataflow" Version="4.6.0" />
     <PackageReference Include="System.Reflection.TypeExtensions" Version="4.3.0" />
-  </ItemGroup>
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp1.0' ">
-    <PackageReference Include="System.Runtime.Loader" Version="4.3.0" />
-  </ItemGroup>
-  <ItemGroup>
     <PackageReference Include="AsyncUsageAnalyzers" Version="1.0.0-alpha003" PrivateAssets="all" />
     <PackageReference Include="Microsoft.VisualStudio.Validation" Version="15.0.82" />
     <PackageReference Include="System.Composition" Version="1.0.31" />
     <PackageReference Include="System.Reflection.Emit" Version="4.3.0" />
     <PackageReference Include="System.Reflection.Metadata" Version="1.3.0" />
     <PackageReference Include="Roslyn.Diagnostics.Analyzers" Version="1.2.0-beta2" PrivateAssets="all" />
   </ItemGroup>
-  <ItemGroup Condition=" '$(TargetsDesktop)' == 'true' ">
+  <ItemGroup>
     <AdditionalFiles Include="PublicAPI.Shipped.txt" />
```

Comment:
Was a little surprised these files didn't seem to change in this PR despite the deletions and typeforwards, but if that's right, great!

Reply:
The type forwards were in a different assembly, so they wouldn't be picked up by this project's api txt files.